src/style.css

```diff
@@ -1 +1,66 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf) format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 900;
+  font-style: normal;
+}
+
+.search__form {
+  margin-top: 20px;
+}
+
+.search__input {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Add more fallback fonts to increase the likelihood that some font will be available, for example: `font-family: Avenir, Arial, Helvetica, sans-serif, Verdana;`